config.toml

```diff
@@ -62,41 +62,34 @@ name = 'Ensign'
 url = 'https://rotational.app'
 weight = 1
 
-[[Languages.en.menu.main]]
-identifier = 'playground'
-pageRef = 'playground'
-name = 'Data Playground'
-url = '/data-playground'
-weight = 2
-
-[[Languages.en.menu.main]]
-identifier = 'opensource'
-pageRef = 'opensource'
-name = 'Open Source'
-url = '/opensource'
-weight = 3
-
 [[Languages.en.menu.main]]
 identifier = 'blog'
 pageRef = 'blog'
 name = 'Blog'
 url = '/blog'
-weight = 4
+weight = 2
 
 [[Languages.en.menu.main]]
 identifier = 'about'
 pageRef = 'about'
 name = 'About'
 url = '/about'
-weight = 5
+weight = 3
+
+[[Languages.en.menu.main]]
+identifier = 'contact'
+pageRef = 'contact'
+name = 'Request a Demo'
+url = '/contact'
+weight = 4
 
 
 [[Languages.en.menu.main]]
 identifier = 'login'
 pageRef = 'login'
-name = 'Login'
+name = 'Get Ensign Free'
```

Comment:
Thank you for making this edit! Can we also change the url to link to https://rotational.app/register ?